### PR TITLE
[semi-modular] a few misc clothing items

### PR DESCRIPTION
**Pull request by werewolfboyfriend**
## About The Pull Request
Adds a small collection of clothes:
Cloth footwraps (digi-friendly), a bunny suit, and hazard jumpsuits for atmos tech and CE.
Bunny suit and foot wraps salvaged from the old, long-defunct Eros Research Platform codebase.
They're all available in loadout, and also added the normal engineer hazard jumpsuit to loadout. Footwraps + existing leg wraps are also added to the clothesmate and the atmos suit is in atmosdrobe.
The hazard jumpsuits are the regular job jumpsuits from Bee Station, but they resemble the hazard jumpsuits here more so.
I also added the source engine style error sprites to the clothing files. some of 'em had no error icons at all, so now they do.
Screenshot 2021-04-26 013236

## Why It's Good For The Game
customization n all that

## Changelog
:cl:
add: Hazard jumpsuits for the chief engineer and atmos tech are now available in the loadout. Dorms-time vendors are now stocking bunny suits. Additionally, plain cloth footwraps are now available for both plantigrade and digitigrade crew.
fix: Added proper missing icon sprites for skyrat clothing.
/:cl:

### CODE REVIEW

**Comment by thestubborn** [comment]
i found some bunny costumes in another coderbase, want em?

File: code/modules/vending/clothesmate.dm [hunk 1]
```diff
@@ -137,6 +137,13 @@
 		            /obj/item/clothing/shoes/cowboy = 2,
 		            /obj/item/clothing/shoes/cowboy/white = 2,
 		            /obj/item/clothing/shoes/cowboy/black = 2,
+		            //SKYRAT EDIT: ADDITION: START - Footwraps
```
Comment (NullDagaf) [inline code/modules/vending/clothesmate.dm]: I thought it's BEGIN instead of START..?

Comment (werewolfboyfriend) [reply code/modules/vending/clothesmate.dm]: Everything in the clothesmate uses start, so I just... went with it.  🤷 I can change them all to begin if preferred

**Comment by werewolfboyfriend** [comment]
> i found some bunny costumes in another coderbase, want em?

Sure

**Comment by thestubborn** [comment]
> 
> 
> > i found some bunny costumes in another coderbase, want em?
> 
> Sure

whats your cord?

**Comment by werewolfboyfriend** [comment]
> > > i found some bunny costumes in another coderbase, want em?
> > 
> > 
> > Sure
> 
> whats your cord?

oh its Bear#4444

File: code/modules/vending/clothesmate.dm (hunk 1 above)
Comment (NullDagaf) [reply code/modules/vending/clothesmate.dm]: Less about "preferred", more just confused about how is it actually supposed to be.

**Comment by werewolfboyfriend** [comment]
added white bunny suit, courtesy of @thestubborn 
I edited it to remove the legs, since underclothes can work for that.
Screenshot 2021-04-27 234401

**Comment by thestubborn** [comment]
"code/modules/vending/clothesmate.dm:					 // SKYRAT EDIT BEGIN - Adds polychromic gear to ClothesMates
mixed <tab><space> indentation detected"

**Pull request closed without merging**